controllers/thoughtController.js

```diff
@@ -0,0 +1,132 @@
+const { json } = require("express");
+const { Thought } = require("../models/Thought");
+const User = require("../models/User");
+
+module.exports = {
+  async getThoughts(req, res) {
+    try {
+      const thoughts = await Thought.find();
+      res.json(thoughts);
+    } catch (err) {
+      res.status(500).json(err);
+    }
+  },
+  async getSingleThought(req, res) {
+    try {
+      const thought = await Thought.findById(req.params.thoughtId).select(
+        "-__v"
+      );
+      if (!thought) {
```

Comment:
really nice handling if the thought doesn't exist, this kind of holistic approach is great to carry through into future work :)